<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="courseCreationGroupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Course Creation - Course Details</string>
   </property>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="courseNumberLabel">
         <property name="text">
          <string>Course Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="courseNameLabel">
         <property name="text">
          <string>Course Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="courseSectionLabel">
         <property name="text">
          <string>Course Section</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="courseSemesterLabel">
         <property name="text">
          <string>Course Semester</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="courseNumberField"/>
       </item>
       <item>
        <widget class="QLineEdit" name="courseNameField"/>
       </item>
       <item>
        <widget class="QLineEdit" name="courseSectionField"/>
       </item>
       <item>
        <widget class="QLineEdit" name="courseSemesterField"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
